init-server.h

```diff
@@ -18,47 +16,108 @@ void printSocketBuff() {
 bool readSocket(WiFiClient& client, uint8_t dataType) {
   bool res = false;
   uint16_t msgLength = client.available();
-  
-  if(msgLength > 0 && dataType == requestHeader) {
+
+  if (msgLength > 0 && dataType == requestHeader) {
     memset(socketBuff, '\0', MAX_SOCKET_BUFF_SIZE);
     char* tmp = nullptr;
```

Comment:
I think that would be nice to define this variable on stack - char tmp[msgLen]. Try to replace old definition with this one(May not work, but at least we need to try). To test this approach - remove also lines 22 and 27-28

Reply:
We need much more refactoring here and in another place
Let us meet to discuss that and let it be a part of another pull request